projects/samples/robotbenchmark/include/tabs.css

```diff
@@ -0,0 +1,64 @@
+/* Taken from https://codepen.io/Wendy-Ho/pen/MWWBvmd */
+body {
+  font-family: arial, sans-serif;
+  font-size: 10pt;
+}
+
+p, h2, h3, code {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+.wrapper{
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 5px;
+}
+.tab{
+  cursor: pointer;
+  padding:10px 20px;
+  margin:0px 2px;
+  background:#000;
+  display:inline-block;
+  color:#fff;
+  border-radius:3px 3px 0px 0px;
+  box-shadow: 0 0.5rem 0.8rem #00000080;
+}
+.panels{
+  background:#fffffff6;
+  box-shadow: 0 2rem 2rem #00000080;
+  min-height:200px;
+  width:100%;
+  max-width:500px;
+  border-radius:3px;
+  padding:20px;
+}
+.panel{
+  display:none;
+  animation: fadein .8s;
+}
+@keyframes fadein {
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+.radio{
+  display:none;
+}
+#one:checked ~ .panels #one-panel,
+#two:checked ~ .panels #two-panel,
+#three:checked ~ .panels #three-panel,
+#four:checked ~ .panels #four-panel{
+  display:block
+}
```

Comment:
Is not possible to use a class instead of ids?
That would be more generic and scalable.